api/handlers/job_handler.go

```diff
@@ -30,29 +30,29 @@ func (this *JobHandler) Routes() *restful.WebService {
 
 	service.Route(service.GET("/").
 		Filter(basicAuthenticate).
-		To(this.ListJobs).
+		To(this.SelectAll).
 		Doc("List all Jobs").
 		Returns(200, "OK", []models.Job{}))
 
 	service.Route(service.GET("{id}").
 		Filter(basicAuthenticate).
-		To(this.GetJob).
+		To(this.SelectByID).
 		Doc("Return a single Job").
 		Param(id).
 		Writes(models.Job{}))
 
 	service.Route(service.DELETE("/{id}").
 		Filter(basicAuthenticate).
-		To(this.DeleteJob).
+		To(this.Delete).
 		Doc("Stop and remove a job").
 		Param(id).
 		Returns(http.StatusNoContent, "Deleted", nil))
 
 	return service
 }
 
-func (this *JobHandler) ListJobs(request *restful.Request, response *restful.Response) {
-	jobs, err := this.JobLogic.ListJobs()
+func (this *JobHandler) SelectAll(request *restful.Request, response *restful.Response) {
```

Comment:
[nit] Since we're touching JobHandler methods, would we want to update `this` to be more Golangish in this PR?